backend/services/API.js

```diff
@@ -31,25 +39,32 @@ async function getSearchedImages (apiType, limit, searchTerm, offset) {
  * Gets trending gifs/images with limit
  */
 async function getTrendingData (apiType, limit, offset) {
```

Comment:
I think this function is getting complicated and harder to read, considering we will be adding more APIs. We should split this per API and use it within its routes folder or somewhere else. What do you think ?

Reply:
yes, I think this will be good.
We can make different functions for search and trendy images in the respective file of different APIs which are inside the routes folder.
Should I make the required changes for that ?
 Than we can delete this services folder because after we implement the functions in their respective file, we no longer need this folder.

Reply:
Sounds great to me! It would be amazing if you could do that. Or if you want we can iterate this over another story. As you wish.

Reply:
I will do it. No problem!👍🏻